### DISCUSSION TITLE
AI 時代的版權之爭

### DISCUSSION THREAD

**Discussion by Tina-Peng-1566** [1101-GS4719-C/Python-Community-GS4719-C#2048] [General]
https://technews.tw/2024/01/09/ai-era-copyright/
AI 時代的版權之爭：《紐約時報》對抗 OpenAI 和微軟
https://technews.tw/2024/05/24/ai-and-media-copyright-defense-battle/
誰擁有新聞？AI 與媒體的版權保衛戰	

年初，多家媒體對OpenAI和微軟提告，指控這兩家公司未經授權就大量使用了他們的新聞來開發 ChatGPT 與 Copilot 等系統，有嚴重的版權侵權行為。這也衍生出了「為了研發AI而忽略新聞的版權」還是「為了顧及新聞版權而降低AI進步的速度」的兩難的情況。

自從生成式AI出現，除了擔心未來某些工作會被AI取代，也質疑AI生成內容的真實性，所以，為了訓練AI而「餵」給它的資料庫內容就特別重要--訓練用的資料必須是真實無錯的，否則AI可能就會模仿真正記者的口吻「一本正經的胡說八道」，讓人無法分辨，進而散佈錯誤訊息。

我認同新聞的版權，但作為使用者，我自然會希望AI越進步、能幫助到人類越多越好，所以我很樂見英國權威財經媒體《金融時報》等多家傳統媒體與AI公司合作，化解了版權糾紛。策略夥伴的合作關係，更能創造三贏--AI持續進步且提升可信度，媒體業者獲得相應的報酬，使用者也確切得到AI和正確資訊的幫助。

**Comment by slps101023** [comment]
確保數據來源的合法性與真實性：授權合作可讓 AI 使用更高品質的內容進行訓練，降低生成錯誤資訊的風險。
讓媒體獲得收入：透過授權協議，媒體機構能從 AI 的發展中分得一杯羹，減少因技術進步帶來的損失。
提升 AI 的可信度與影響力：有了權威數據支持，生成內容的準確性與公信力都會提高，更能幫助使用者解決實際問題。
